### PR TITLE
docs: add info on crash logs

### PR DESCRIPTION
**Pull request by brianlheim**
Purpose and Motivation
----------------------

many issue reports could be made better if we had crash logs/reports.

Types of changes
----------------

this PR adds info on crash reports and how to generate them, and prompts
reporters for one in the GitHub issue template.

- Documentation (non-code change which corrects or adds documentation for existing features)

Checklist
---------

- [x] Updated documentation, if necessary
- [x] This PR is ready for review

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -138,6 +141,34 @@ Rebasing is one way to integrate changes from one branch onto another, in this c
...
+
+    pkill -SIGSEGV <sc-executable> # may need to execute twice to force a crash
+
+The crash log will be placed in `~/Library/Logs/DiagnosticReports` with the others.
+
+### Linux
+
+A core dump file is generated when an application crashes. See [this helpful article](https://linux-audit.com/understand-and-configure-core-dumps-work-on-linux/) for information on core dumps and how to enable them on your machine. You don't need to send us the full core dump (it will probably be quite large), but you can generate a helpful backgrace with gdb:
+
+    gdb <sc-executable> <core-file> -ex where -ex exit
```
Comment (patrickdupuis) [inline CONTRIBUTING.md]: Are you certain that `exit` is correct here? Did you mean to write `-ex quit`?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ Error message (for bugs)
 // Please paste any error messages here **in their entirety**.
 // If this is a SuperCollider error message, include the full stack trace.
 // Link to a Gist (https://gist.github.com) if the message is long.
+// If your issue involves a crash, add a crash report. See CONTRIBUTING.md
+// in the root folder of the repository for instructions.
```
Comment (patrickdupuis) [inline .github/ISSUE_TEMPLATE.md]: I'd move `See CONTRIBUTING.md` down to the next line.

File: CONTRIBUTING.md (hunk 1 above)
Comment (brianlheim) [reply CONTRIBUTING.md]: yep, thanks for catching that!

**Comment by brianlheim** [comment]
@patrickdupuis addressed changes, thanks for the review!

**Pull request merged**